index.html

```diff
@@ -33,21 +33,31 @@
     };
   });
 
+const trusted = /^127\.0\.0\.1(?::|$)|localhost(?::|$)|(?:app\.|local\.)?webaverse\.com|webaverse\.link/
+const github = /githubpreview\.dev$/
+
 const _message = e => {
   const url = new URL(e.origin);
-  if (url && /^127\.0\.0\.1(?::|$)|localhost(?::|$)|(?:app\.|local\.)?webaverse\.com|webaverse\.link/.test(url.host)) {
+  const gh = github.test(url.host)
+  const allowed = trusted.test(url.host) || gh
```

Comment:
Semi style. It's not in this repo but the style file is here: https://github.com/webaverse/app/blob/master/.eslintrc.json